elytron/src/test/resources/org/wildfly/extension/elytron/mycredstore.cs

```diff
@@ -0,0 +1,2 @@
+# Properties Credential Store (Do Not Modify)
+key=RUxZAUsFr8w4PAFmH+p5ExrjHRhgQV6Ij5cSp/oNVnL2rptfWg==
```

Comment:
I think it should be possible to generate this dynamically during the test.

Reply:
I need this one to be manually included because I need to know the secret key to decode the identities and can't do that unless I statically create it before the test.
Although I could hardcode the secret-key string and generate the secret key during the test instead.